web/home.go

```diff
@@ -13,8 +13,9 @@ type HomeData struct {
 
 func HomeHandler(c *gin.Context) {
 	data := HomeData{
-		Title:     defaultLayoutData.Title,
-		Paragraph: "This is the home page",
+		Title: defaultLayoutData.Title,
```

Comment:
As commented, I think the `Title` and `Paragraph` should go in the html template itself, together with everything else, but well, who cares, we will touch this many time on the road hehe

Reply:
The title is also rendered in the menu so I'll leave it, but I see no point in keeping the paragraph there. I'll put it in the template. Thanks!